src/test/java/andreeaG/tests/tema8/pageTests/DisabledDynamicButtonsTest.java

```diff
@@ -0,0 +1,20 @@
+package andreeaG.tests.tema8.pageTests;
+
+import andreeaG.tests.tema8.pageObjects.DisabledDynamicButtonsPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DisabledDynamicButtonsTest extends BaseTest {
+    @Test(groups = {"mobile", "desktop"})
+    public void verifyClickButtonsInOrder() {
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-disabled.html");
+        DisabledDynamicButtonsPage disabledDynamicButtonsPage = new DisabledDynamicButtonsPage(driver);
+        disabledDynamicButtonsPage.setStartButton();
+        disabledDynamicButtonsPage.clickStartButton();
+        disabledDynamicButtonsPage.clickOnOneButton();
+        disabledDynamicButtonsPage.clickOnSecondButton();
+        disabledDynamicButtonsPage.clickOnThirdButton();
+        Assert.assertEquals(disabledDynamicButtonsPage.messageDisplay(), "Click Buttons In Order");
```

Comment:
You should assert for "All Buttons Clicked" and not "Click Buttons In Order". After clicking on buttons, wait for the invisibility of "Wait ..." as you did for SimpleDynamicButtonsTest. Thanks!